### ISSUE TITLE
Typeahead a11y issues within Firefox

### ISSUE THREAD

**Issue by daniiilponomarev** [angular-ui/bootstrap#6726]
Discovered a few bugs with accessibility in Firefox 62.0.2.
* Type 2 symbols in a search field. First option of dropdown is focused but NVDA doesn't read it until we begin to select other options with arrows buttons.
* Type the third symbol. First option is focused but isn't read (even within Chrome 71).
* After typing the third symbol NVDA cannot count number of options correctly. For example, with "al" we have 3 options, with "ala" - 2 options, but each of them is read as "<name> <number> of three". Deleting symbols with changing number of options also causes the same problem.

**Comment by icfantv** [comment]
Closing this as this project is no longer being maintained, per the README and the ISSUE_TEMPLATE.
